docs/_posts/2016-01-11-installation-guide.md

```diff
@@ -155,9 +155,13 @@ Append the following line to `$CATALINA_HOME/bin/setenv.sh` or, if using Tomcat
 
 (Update the security method if you are using LDAP or Active Directory LDAP.)
 
-Create the directory `/storage/miso`:
+Create the directory `/storage/miso` and subdirectory `/storage/miso/logs`. 
 
-  	cd /storage/miso
+  	mkdir -p /storage/miso/logs
+    
+If you have created a group for your Tomcat server it will need write permission to the storage directory.
```

Comment:
Since this can vary from system to system, please reword to "Ensure that the user that Tomcat runs as has write permission to the storage directory. For example, if the user is in the 'tomcat' group:"